Dockerfile

```diff
@@ -17,10 +17,12 @@ RUN \
     curl -sL https://deb.nodesource.com/setup_10.x | bash -; \
     apt-get install -y nodejs; \
     cd kinto/plugins/admin; npm install; npm run build; \
+#    pip3 install --upgrade pip \
```

Comment:
So, we upgrade or not?

Reply:
Yeah, buddy, why not?! Actually, I should have said that obviously, it has to be uncommented.